src/data.json

```diff
@@ -0,0 +1,407 @@
+      "common_names": ["Thyme"],
+      "description": "Thyme is a perennial evergreen herb in the Lamiaceae (Mint) family. It is related to Oregano. Thyme is cold-hardy to -28.8 °C and a perennial in US Hardiness zones 5-8. Thyme can live for 5-6 years but needs to be divided or replaced after 3-4 years. It can be used fresh or dried to season dishes.",
+      "sun_requirements": "Full Sun",
+      "sowing_method": "Transplant cuttings or seedlings or direct seed indoors",
+      "spread": 15,
+      "row_spacing": 30,
+      "height": 25,
+      "processing_pictures": 0,
+      "guides_count": 0,
+      "main_image_path": "https://s3.amazonaws.com/openfarm-project/production/media/pictures/attachments/58e32ad6ca5bf60004000164.jpg?1491282642",
+      "taxon": "Species"
+    }
+  },
+  {
+    "type": "crops",
+    "id": "54bda00e3961370003150400",
+    "attributes": {
+      "name": "Tiny Tim Tomato",
+      "slug": "tiny-tim-tomato",
+      "binomial_name": "Solanum lycopersicum",
+      "common_names": ["Tomato", "Tomate"],
+      "description": "The tomato is the fruit of the tomato plant, a member of the Nightshade family (Solanaceae). The fruit grows on a small compact bush.",
+      "sun_requirements": "Full Sun",
+      "sowing_method": "Direct seed indoors, transplant seedlings outside after hardening off",
+      "spread": 45,
+      "row_spacing": 45,
+      "height": 45,
+      "processing_pictures": 0,
+      "guides_count": 37,
+      "main_image_path": "https://s3.amazonaws.com/openfarm-project/production/media/pictures/attachments/5dc3618ef2c1020004f936e4.jpg?1573085580",
+      "taxon": "Species"
+    }
+  },
+  {
+    "type": "crops",
+    "id": "543998a83162320002000000",
+    "attributes": {
+      "name": "UF Micro Tom Tomato",
+      "slug": "uf-micro-tom-tomato",
+      "binomial_name": "Solanum lycopersicum var. cerasiforme",
+      "common_names": [],
+      "description": "Cherry tomatoes range in size from a thumbtip to the size of a golf ball. Their shape ranges from spherical to slightly oblong to pointed at the bottom. They are often red, but can also be yellow, green, striped, and even black. More oblong cherry tomatoes often share characteristics with plum tomatoes, and are known as grape tomatoes. Cherry tomatoes can be quite sweet (such as the Sungold or Sunsweet yellow varieties), more traditionally acidic, or deep in flavor. They are delicious for snacking, in salads, lightly roasted or grilled, or baked. They require little to no pruning unlike larger tomato plants.",
+      "sun_requirements": "Full Sun",
+      "sowing_method": "Sow seeds indoors 6-8 weeks before the last expected frost",
+      "spread": 45,
+      "row_spacing": 60,
+      "height": 60,
+      "processing_pictures": 0,
+      "guides_count": 14,
+      "main_image_path": "https://s3.amazonaws.com/openfarm-project/production/media/pictures/attachments/57073ec74a3af70003000009.jpg?1460092612",
+      "taxon": "Species"
+    }
+  }
+]
```

Comment:
so i guess, you didnt write this on your own. otherwise, i would be very impressed 😄